modules/instance_template/versions.tf

```
@@ -17,9 +17,9 @@
 terraform {
   required_version = ">=0.13.0"
   required_providers {
-    google = {
-      source  = "hashicorp/google"
-      version = ">= 4.67, < 6"
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 5.5"
```

Recommend if using `>=` that we still include `< 6` to avoid breaking change in the future major release.

```suggestion
      version = ">= 5.5, < 6"
```